src/pages/GroupChatNameEditPage.tsx

```diff
@@ -66,13 +70,13 @@ function GroupChatNameEditPage({groupChatDraft, route}: GroupChatNameEditPagePro
         (values: FormOnyxValues<typeof ONYXKEYS.FORMS.NEW_CHAT_NAME_FORM>) => {
             if (isUpdatingExistingReport) {
                 if (values[INPUT_IDS.NEW_CHAT_NAME] !== currentChatName) {
-                    Report.updateGroupChatName(reportID, values[INPUT_IDS.NEW_CHAT_NAME] ?? '');
+                    ReportActions.updateGroupChatName(reportID, values[INPUT_IDS.NEW_CHAT_NAME] ?? '');
```

Comment:
I would rather have this stay `Report` and update the type name to `ReportOnyxType` or something. `ReportActions` is easy to confuse with `reportActions`.

Reply:
Makes sense! Updated!